components/helper/ProfileInfo.tsx

```diff
@@ -0,0 +1,82 @@
+import { Card, Divider, Link, Typography } from '@mui/material';
```

Comment:
let's not use the same component for team member and for prospect. For team members we care about different informations and long term the information displayed on their page will differ a lot which will make the implementation ugly.